modules/monitoring-granting-users-permissions-for-core-platform-monitoring.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/configuring-the-monitoring-stack.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="granting-users-permissions-for-core-platform-monitoring_{context}"]
+= Granting users permissions for core platform monitoring
+
+As a cluster administrator, you can monitor all core {product-title} and user-defined projects.
+
+You can also grant developers and other users different permissions for core platform monitoring:
+
+* To monitor core platform monitoring components.
+* To access the Thanos Querier API endpoints.
+* To manage alert silences.
+* To monitor the Alertmanager API.
+
+You can grant the permissions by assigning one of the following monitoring roles and cluster roles:
+
+|===
+|Name |Description |Project
+
+|`cluster-monitoring-metrics-api` 
+|Users with this role have the ability to access Thanos Querier API endpoints. Additionally, it grants access to user-defined Prometheus and Thanos Ruler API endpoints. 
+|`openshift-monitoring`
+
+|`cluster-monitoring-operator-alert-customization` 
+|Users with this role can manage `AlertingRule` and `AlertRelabelConfig` resources for core platform monitoring. These permissions are required for the alert customization feature. 
+|`openshift-monitoring`
+
+|`monitoring-alertmanager-edit` 
+|Users with this role can manage alert silences for core platform monitoring in the *Administrator* perspective of the {product-title} web console. Additionally, it provides access to the Alertmanager web server on port 9094. 
+|`openshift-monitoring`
+
+|`monitoring-alertmanager-view` 
+|Users with this role can monitor the Alertmanager API for core platform monitoring. They can also view alert silences in the *Administrator* perspective of the {product-title} web console. 
```

Comment:
the wording isn't consistent with 'monitoring-alertmanager-edit'. Should we fix?